templates/4b_normalize_image_data.ipynb

```diff
@@ -9,7 +9,9 @@
    "source": [
```

Comment:
<p>Changes in detector sensitivity can result in images with different intensities, in the absence of any true difference in biological signal. </p><p>We use MPH as a measure of detector sensitivity to determine when this has happened. </p><p>We use the tuning curve generated in 1 to map changes in MPH to changes in sensitivity, to determine how much of an adjustment needs to be made</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>